### ISSUE TITLE
Linear Regression

### ISSUE THREAD

**Issue by kanikamittal1811** [DeepConnectAI/Self_Paced_Learning#7]
As a contributor 
- explain Linear Regression basics by implementing python code from scratch
- visualize Regression line and Loss Curve
- show the effects of changing hyperparameters
- Use L1 and L2 Regularization to solve Overfitting issues

**Comment by abhishek-mishra17** [comment]
@kanikamittal1811  i would love to work on this.

**Comment by pradyumnjain** [comment]
@abhishek-pes You can start by creating resources mentioned in the above guidance. They are not to be strictly followed though. Make resources using code snippets that best explain the important concepts related to the given topic.

**Comment by abhishek-mishra17** [comment]
@pradyumnjain  thanks a lot for assigning me this, I will start working on this and create a pull request as soon as possible

**Comment by pradyumnjain** [comment]
@abhishek-pes  great!
